lte/gateway/c/oai/lib/openflow/controller/GTPApplication.cpp

```diff
@@ -228,7 +228,7 @@ static void add_downlink_match_ipv6(
   /* TODO-Made this local fix as it was not yet available in master
    * without this fix ovs rules are not getting created
    */
-  of13::EthType ip6_type(0x086DD);
+  of13::EthType ip6_type(0x86DD);
```

Comment:
why is this reverted?

Reply:
It should be 0x86DD and not 0x086DD. 0 is redundant